lib/friends/event.rb

```diff
@@ -64,6 +64,7 @@ def initialize(str: "")
 
     attr_reader :date
     attr_accessor :description
+    attr_accessor :implicit_location
```

Comment:
Looks like this could just be `attr_writer`, right?

Reply:
Ha, I've not always been super clear on the difference, but I just played around with `irb` and I understand now! Totally, it's only called once outside of `Event` to be assigned, so I can change this 👍